test/pageobjects/Welcome.js

```diff
@@ -51,6 +51,7 @@ class Welcome extends BasePage {
   }
 
   async continueToNextStep() {
+    this.driver.sleep(100)
```

Comment:
I think there should be a note here about why this explicit `sleep` is needed here.